### PR TITLE
Allow hiding annotations in the UI

### LINKED ISSUE #169: Option to remove summary or description
Hey there - thanks for `unsee`, just started using and it's great.

One question: is it possible to remove either alert `description` or `summary` from the dashboard? Would clear up a good amount of dashboard space. For our purposes we don't need both.

Thanks!

### PR DESCRIPTION
**Pull request by prymitive**
See `README` changes for details.
Fixes #169

hidden

### CODE REVIEW

File: assets/templates/alertgroup.html [hunk 1]
```diff
@@ -308,3 +312,14 @@
     <%- label.text %>
   </<%= elem %>>
 </script>
+
+<script type="application/json" id="alert-annotation">
+<% var cls = "" %>
```
Comment (terinjokes) [inline assets/templates/alertgroup.html]: Why not use the `<details>` and `<summary>` elements?

```html
<details open>
  <summary>Summary</summary>
  <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
</details>
```

File: README.md [hunk 2]
```diff
@@ -191,6 +191,62 @@ This option can also be set using `-alertmanager.uris` flag. Example:
 
 This variable is required and there is no default value.
 
+#### ANNOTATIONS_DEFAULT_HIDDEN
```
Comment (terinjokes) [inline README.md]: Wouldn't it make more sense for these to be user-specific options configured in the UI?

Comment (prymitive) [inline README.md]: Probably, but that would be an ugly UI if I dump it under current settings menu, so I'm not even going to try right now.

File: assets/templates/alertgroup.html (hunk 1 above)
Comment (prymitive) [inline assets/templates/alertgroup.html]: `cls` is used to set `hidden hidden-annotation` if needed, not sure how `<details>` and `<summary>` relates to that

**Pull request merged**